int32 NAO
---
int32 hand
